regressionTests.cmake

```diff
@@ -1155,3 +1155,10 @@ add_test_compareECLFiles(CASENAME 02_wgrupcon
                          ABS_TOL ${abs_tol}
                          REL_TOL ${rel_tol}
                          DIR wgrupcon)
+add_test_compareECLFiles(CASENAME 01_vappars
+                         FILENAME 01-VAPPARS
```

Comment:
I don't see a case named `01-VAPPARS`, but we do have a case named [`VAPPARS-01`](https://github.com/OPM/opm-tests/blob/e29c0fa7cac292fedf0f3be49c606c210f5007c4/vappars/VAPPARS-01.DATA) in the `vappars` directory of OPM-Tests.  Did you mean that one?

Reply:
thanks @bska, my bad. This is fixed now.